app/boards/makerbase/mks_canable_v20/mks_canable_v20.conf

```diff
@@ -0,0 +1,20 @@
+# The three lines below are needed to reduce ROM usage
+CONFIG_LTO=y
+CONFIG_SIZE_OPTIMIZATIONS=y
+CONFIG_ISR_TABLES_LOCAL_DECLARATION=y
+
+CONFIG_USB_DEVICE_GS_USB_TIMESTAMP=y
+CONFIG_USB_DEVICE_GS_USB_TERMINATION=n
+CONFIG_USB_DEVICE_GS_USB_MAX_CHANNELS=1
+CONFIG_USB_DEVICE_GS_USB_TIMESTAMP_SOF=y
```

Comment:
Please leave this up to the user to enable. SoF events are hard on the CPU usage and the cost of this added timestamp precision should be evaluated by the user based on their needs, not enabled by default on a board-by-board basis.